opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -280,16 +300,75 @@ def export(
                     UNIQUE_IDENTIFIER_KEY
                 ] = self.unique_identifier
 
-            point = series.points.add()
-            if isinstance(record.aggregator, SumAggregator):
-                data_point = record.aggregator.checkpoint
-            elif isinstance(record.aggregator, ValueObserverAggregator):
-                data_point = record.aggregator.checkpoint.last
+            point_dict = {"interval": {}}
+
+            if isinstance(record.aggregator, HistogramAggregator):
+                count = sum(record.aggregator.checkpoint.values())
+                buckets = list(record.aggregator.checkpoint.items())
+
+                point_dict["value"] = {
+                    "distribution_value": Distribution(
+                        count=count,
+                        bucket_counts=[bucket[1] for bucket in buckets],
+                        bucket_options={
+                            "explicit_buckets": {
+                                # don't put in > bucket
+                                "bounds": [
+                                    bucket[0] for bucket in buckets[:-1]
+                                ]
+                            }
+                        },
+                        # exemplars=[
```

Comment:
please remove

Reply:
added comment, I would prefer for this to stay unless it really needs to go